examples/210-embedding_models.ipynb

```diff
@@ -0,0 +1,1128 @@
+{
```

Comment:
<blockquote>That features</blockquote><p>Isn't it better to write "these" instead of "that"?</p><p> </p><blockquote>And should be used as lags for future</blockquote><p>May be it's better to write "and should be used in forecasting after applying lag transformation"</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/etna-team/etna/pull/304/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
> Isn't it better to write "these" instead of "that"?

May be

> May be it's better to write "and should be used in forecasting after applying lag transformation"

Here it is unclear for me
- What lag transform is applied for
- When  lag transform is applied - before or after generating features

Reply:
<p>Ok, let's rewrite it like this: "These features aren't regressors and lag transfomation should be applied to them before using in forecasting".</p>